test/interactive_tarantool.lua

```diff
@@ -224,6 +224,14 @@ function mt.close(self)
     self.ph:close()
 end
 
+function mt.repl(child, command, expected)
+    child:execute_command(command)
+    local response = child:read_response()
+    if (expected ~= nil) then
+        t.assert_equals(response, expected)
+    end
+end
```

Comment:
Nits:

1. `child` -> `self` (for consistency with other methods).
2. Extra parentheses in the if-condition (please, add it into your self-check list).

Reply:
Fixed. I also renamed `repl()` to `roundtrip()` as you suggested and added a comment.

Reply:
Please, reflect the method name change in the commit message.

The `response` may be a table, so your initial idea to use `t.assert_equals()` makes sense. I would return it back.

Reply:
My bad, missed that. Fixed.